exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/FragmentWrapper.java

```diff
@@ -35,6 +38,10 @@
 public class FragmentWrapper {
   private final MajorFragmentProfile major;
   private final long start;
+  private final Locale currentLocale = Locale.getDefault();
+  private final String pattern = "dd-MMM-yyyy HH:mm:ss";
+  private final SimpleDateFormat simpleDateFormat = new SimpleDateFormat(
```

Comment:
Oh. I thought the locale did that. But, as long as these timestamps are aligned with the timestamps on the server (where the logs reside), this should be fine, right? Otherwise, this is useful for only _running_ queries and we can skip it altogether. What do you suggest, @paul-rogers ?